visual-dotnet/SauceLabs.Visual/Utils/AuthenticatedGraphQLRequest.cs

```diff
@@ -18,12 +18,12 @@ internal class AuthenticatedGraphQLRequest : GraphQLHttpRequest
             IGraphQLJsonSerializer serializer)
         {
             var message = base.ToHttpRequestMessage(options, serializer);
-            if (string.IsNullOrEmpty(Username) || string.IsNullOrEmpty(AccessKey))
+            if (string.IsNullOrEmpty(Username?.Trim()) || string.IsNullOrEmpty(AccessKey?.Trim()))
```

Comment:
how about creating an extension method for string to check if string is null or empty or white space?

Reply:
Cannot make it an extension as the string may be null (and extension needs an instances).
Created `StringUtils.IsNullOrEmpty` to check for this.